### PR TITLE
use ETH instead of wETH on exchange page

### PR DESCRIPTION
**Pull request by LeifuChen**
## Description
This PR fixes the inconsistent use of wording 'ETH' in en.json and GetL2GasCard MessageButton, which affect the Kwenta exchange page. Additionally, Kwenta should prompt users to "Bridge ETH" instead of "Buy ETH".

## Related issue
#282 

## Motivation and Context
Kwenta should use the word 'ETH' consistently in UI and prompt users to "Bridge ETH".

## How Has This Been Tested?
This PR only changes texts so it won't break any test.

## Screenshots (if appropriate):
kwenta-bridge-eth

### CODE REVIEW

**Comment by LeifuChen** [comment]
@platschi The change has been made. Now the text of the button shows "Bridge ETH". Thanks.

**Comment by LeifuChen** [comment]
@platschi @koredefashokun Shall we correctly use two different i18nKey to show "Buy Synths" and "Bridge ETH" in different situtations?

File: translations/en.json [hunk 1]
```diff
@@ -637,6 +637,7 @@
 			"currency-price": "<0>{{currencyKey}}</0> Price",
 			"synthetic-currency-name": "Synthetic {{currencyName}}",
 			"trade-to-currency": "Trade to <0>{{currencyKey}}</0>",
+			"bridge-currency": "Bridge <0>{{currencyKey}}</0>",
 			"buy-currency": "Buy <0>{{currencyKey}}</0>",
```
Comment (JChiaramonte7) [inline translations/en.json:641]: I think you can drop `buy-currency` if it's not used anywhere else.

Comment (platschi) [reply translations/en.json:641]: It's being used in https://github.com/Kwenta/kwenta/blob/10f59cd74aff8f556bc5658103bb7e84493a8db2/sections/exchange/FooterCard/NoSynthsCard/NoSynthsCard.tsx#L58

**Review by JChiaramonte7 — APPROVED** [review]

**Pull request merged**